File: Modelo/Venta.py
class Venta:
    def __init__(self):
        self._id = 0
        self._items = []
        self._total = 0.0
        self._descuento = 0.0
        self._metodoPago = ''
        self._fecha = None
        self._cajero = ''

    def getId(self):
        return self._id

    def setId(self, id):
        self._id = id

    def getItems(self):
        return self._items

    def setItems(self, items):
        self._items = items

    def getTotal(self):
        return self._total

    def setTotal(self, total):
        self._total = total

    def getDescuento(self):
        return self._descuento

    def setDescuento(self, descuento):
        self._descuento = descuento

    def getMetodoPago(self):
        return self._metodoPago

    def setMetodoPago(self, metodoPago):
        self._metodoPago = metodoPago

    def getFecha(self):
        return self._fecha

    def setFecha(self, fecha):
        self._fecha = fecha

    def getCajero(self):
        return self._cajero

    def setCajero(self, cajero):
        self._cajero = cajero

File: Modelo/Venta.cs
using System;
using System.Collections.Generic;

namespace punto_de_venta_C_.Modelo
{
    public class Venta
    {
        private int id;
        private List<ItemCarrito> items;
        private double total;
        private double descuento;
        private string metodoPago;
        private DateTime fecha;
        private string cajero;

        public Venta(int id, List<ItemCarrito> items, double total, string metodoPago, DateTime fecha, string cajero)
        {
            this.id = id;
            this.items = items;
            this.total = total;
            this.descuento = 0;
            this.metodoPago = metodoPago;
            this.fecha = fecha;
            this.cajero = cajero;
        }

        public Venta(int id, List<ItemCarrito> items, double total, double descuento, string metodoPago, DateTime fecha, string cajero)
        {
            this.id = id;
            this.items = items;
            this.total = total;
            this.descuento = descuento;
            this.metodoPago = metodoPago;
            this.fecha = fecha;
            this.cajero = cajero;
        }

        public int getId() { return id; }
        public void setId(int id) { this.id = id; }
        public List<ItemCarrito> getItems() { return items; }
        public void setItems(List<ItemCarrito> items) { this.items = items; }
        public double getTotal() { return total; }
        public void setTotal(double total) { this.total = total; }
        public string getMetodoPago() { return metodoPago; }
        public void setMetodoPago(string metodoPago) { this.metodoPago = metodoPago; }
        public DateTime getFecha() { return fecha; }
        public void setFecha(DateTime fecha) { this.fecha = fecha; }
        public string getCajero() { return cajero; }
        public void setCajero(string cajero) { this.cajero = cajero; }
        public double getDescuento() { return descuento; }
        public void setDescuento(double descuento) { this.descuento = descuento; }

        public double getSubtotal()
        {
            return Math.Round((total + descuento) * 100.0) / 100.0;
        }

        public int getTotalItems()
        {
            return items.Count;
        }
    }
}
